### DISCUSSION TITLE
Has Stock Beta been removed from ticker.info?

### DISCUSSION THREAD

**Discussion by mattpabi** [ranaroussi/yfinance#1645] [General]
Hi, I have been looking around for available APIs to get the beta of a given stock.

I initially thought of using `yfinance` to do so, since coming across this question from [Stack Overflow](https://stackoverflow.com/questions/66509788/yfinance-downloading-multiple-parameters-beta-mkt-cap-etc-for-multiple-sto) which clearly shows the users being able to do:

```
info = yf.Ticker('MSFT').info
beta = info['beta']
```

However, this does not seem to be the case anymore since 'beta' is not an available key in `ticker.info` as it returns:

```
info = yf.Ticker('MSFT').info
beta = info['beta']
>>> KeyError: 'beta'
```

Has `beta` been removed from `yfinance`? The beta data is still available on Yahoo's site.

**Comment by XingzheLi123** [comment]
Same question! Compared to just a week ago, a lot of data is now gone from 'info'

**Comment by ValueRaider** [comment]
`yfinance` hasn't done anything. There used to be a bug, but was fixed here #1603. Maybe Yahoo changed their end?

**Reply by mattpabi** [reply]
Oh ok.

Which is a bit weird, given that `yfinance` is able to provide the 52 week high and low data from Yahoo, which should be categorised with `beta` as seen on the site [here](https://finance.yahoo.com/quote/MSFT/key-statistics/).

Is there any way we can check to hopefully re-add `beta` into `info[]`? @XingzheLi123 [also mentioned](https://github.com/ranaroussi/yfinance/discussions/1645#discussioncomment-6624232) some data is now gone after a recent update within this GitHub thread.

**Comment by ValueRaider** [comment]
Continue discussion in Issue #1649
